.github/workflows/canary-integration-test.yml

```diff
@@ -91,6 +91,11 @@ jobs:
         toolbox=$(kubectl get pod -l app=rook-ceph-tools -n rook-ceph -o jsonpath='{.items[*].metadata.name}')
         timeout 60 sh -c "until kubectl -n rook-ceph exec $toolbox -- ceph fs subvolumegroup ls myfs|jq .[0].name|grep -q "group-a"; do sleep 1 && echo 'waiting for the subvolumegroup to be created'; done"
 
+    - name: wait for the rados namespace to be created
+      run: |
+        toolbox=$(kubectl get pod -l app=rook-ceph-tools -n rook-ceph -o jsonpath='{.items[*].metadata.name}')
+        timeout 60 sh -c "until kubectl -n rook-ceph exec $toolbox -- rbd namespace ls replicapool --format=json|jq .[0].name|grep -q "namespace-a"; do sleep 1 && echo 'waiting for the rados namespace to be created'; done"
+
```

Comment:
If we intend to support update/modify and delete operations, we should test that in CI also. It happens too often that a feature's "create" behavior is the only tested one and modify/delete is bugged.

Reply:
added create/delete, there are no parameters to modify for now.

Reply:
What happens if someone modifies the `spec.blockPoolName`? Is that something we intend to allow? How are we verifying the correct behavior (allowed or disallowed change)? 

If we intend not to allow it (seems the right thing to do to me), do we return an error to the user if they change the parameter? We should be checking that in the admission webhook as well if it isn't allowed.

Reply:
Returning error is not useful, as the CR is changed its a problem for delete operation or new rados namespace will get created. admission webhook is the right one. the same change is needed for subvolumegroup also. As it is not an expected one, I will send follow-up PR for both. Hope its fine